wagtail/tests/test_page_queryset.py

```diff
@@ -919,6 +919,23 @@ def test_specific_query_with_annotation(self):
         self.assertEqual(results.first().subscribers_count, 1)
         self.assertEqual(results.last().subscribers_count, 1)
 
+    def test_specific_query_with_alias(self):
+        # #11285
+        # Ensure alias() works with specific() queries.
```

Comment:
Do we actually need a regression test?

Reply:
I think we should always have a unit test for regressions like this. It's helpful, especially for complex cases where reproducing manually is more rare.